2.7.0/done/result_quantization.md

```diff
@@ -27,42 +27,93 @@ The categories below are as follows:
 
 ## quantization
 ### bc breaking
-- [BC-Breaking]Remove capture_pre_autograd_graph references in quantization ([#139505](https://github.com/pytorch/pytorch/pull/139505))
+- Remove capture_pre_autograd_graph references in quantization ([#139505](https://github.com/pytorch/pytorch/pull/139505))
+
```

Comment:
add a header + description here for the APIs going away + mention that it is completing a deprecation. Example https://github.com/janeyx99/torch-release-notes/blob/main/2.7.0/done/result_optim.md#bc-breaking